### PR TITLE
[lldb] Add forward interop expression evaluation tests

### PR DESCRIPTION
**Pull request by augusto2112**
rdar://100284870

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/test/API/lang/swift/cxx_interop/forward/expressions/TestSwiftForwardInteropExpressions.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        self.expect('expr throwException()', substrs=['internal c++ exception breakpoint'], error=True)
+
+        # Check that we can make persistent variables.
+        self.expect('expr var $cxxClass = CxxClass()')
+
+        # Check that we can refer to the persistent variable.
+        self.expect('expr $cxxClass', substrs=['CxxClass', 'a = 100', 'b = 101'])
+
+        # Check that we can call methods on the persistent variable.
+        self.expect('expr $cxxClass.sum()', substrs=['Int32', '201'])
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/cxx_interop/forward/expressions/TestSwiftForwardInteropExpressions.py:51]: Nice.

**Pull request merged**